### PR TITLE
chore: refactor workflows and issues template

### PR DESCRIPTION
**Pull request by sbouchet**
## Was the change discussed in an issue?
fixes #???

### CODE REVIEW

File: .github/workflows/ui_it.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: UI and Integration Tests
+on:
+  workflow_run:
+    workflows: [ "Java CI with Gradle" ]
```
Comment (adietish) [inline .github/workflows/ui_it.yml:4]: @sbouchet: I most likely dont get this fully: 
this workflow should be triggered when the workflow `Java CI with Gradle` is completed. But there isn't a workflow called `Java CI with Gradle`, isn't it?

**Review by adietish — APPROVED** [review]
assuming that I didnt fully understand it, LGTM

File: .github/workflows/ui_it.yml (hunk 1 above)
Comment (sbouchet) [reply .github/workflows/ui_it.yml:4]: it's https://github.com/redhat-developer/intellij-openshift-connector/blob/main/.github/workflows/ci.yml

**Pull request merged**